core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -121,6 +120,239 @@ export class ContributorAdminStatsTable implements OnInit {
     }
   }
 
+  pluralise(num: number, word: string): string {
+    return num === 1 ? word : word + 's';
+  }
+
+  getCountDisplay(
+      count: number,
+      wordCount: number = 0,
+      editDisplay: string = ''): string {
+    return count + this.pluralise(count, ' card') +
+           editDisplay +
+         (
+          wordCount === 0 ?
+                       '' :
+                       ', ' + wordCount + this.pluralise(wordCount, ' word')
+         );
+  }
+
+  getAcceptedSubmittedDisplay(
+      acceptedTranslationCount: number,
+      acceptedTranslationsWithoutReviewerEditsCount: number,
+      acceptedTranslationWordCount: number = 0): string {
+    return this.getCountDisplay(
+      acceptedTranslationCount,
+      acceptedTranslationWordCount,
+      ' (' + acceptedTranslationsWithoutReviewerEditsCount +
```

Comment:
I'd suggest passing the number in and doing all the string concat in getCountDIsplay()

Reply:
Done in getTextForAttribute.